cmake/projects/MySQLCConnector/hunter.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2014-2015, David Linten
+# All rights reserved.
+
+if(DEFINED HUNTER_CMAKE_PROJECTS_MYSQLCCONNECTOR_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_MYSQLCCONNECTOR_HUNTER_CMAKE_ 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    MySQLCConnector
+    VERSION
+    "v6.1.6"
+    URL
+    "https://dev.mysql.com/get/Downloads/Connector-C/mysql-connector-c-6.1.6-src.tar.gz"
```

Comment:
Can you please create pull request will only this change, will merge it. Also please set destination branch to `develop`, see https://github.com/ruslo/hunter/wiki/dev.contribution